modules/dbfs/databaseMock.go

```diff
@@ -138,8 +140,10 @@ func (dm *DatabaseMock) MySQLUserProjects(username string) ([]ProjectMeta, error
 // MySQLProjectCreate is a mock of the real implementation
 func (dm *DatabaseMock) MySQLProjectCreate(username string, projectName string) (int64, error) {
 	dm.FunctionCallCount++
+
+	perm, _ := config.PermissionByLabel("owner")
 	proj := ProjectMeta{
-		PermissionLevel: 10,
+		PermissionLevel: perm.Level,
```

Comment:
Ultimately, this should be moved to the map type as well

Do you think you could add a todo?